src/errors.js

```diff
@@ -0,0 +1,11 @@
+const serverError = (res, err) => {
+  res.statusCode = 500;
+
+  res.setHeader('Content-Type', 'application/json');
+
+  res.message = 'Internal server Error';
```

Comment:
There is a typo in the error message. It should be 'Internal Server Error' instead of 'Internal server Error'.